certif/tests/integration/components/pagination-control_test.js

```diff
@@ -53,4 +56,19 @@ module('Integration | Component | pagination-control', function(hooks) {
     assert.dom('.page-navigation__arrow--previous .icon-button').hasNoClass('disabled');
   });
 
+  test('it should re-route to page with changed page size', async function(assert) {
+    const replaceWithStub = sinon.stub();
+    class RouterStub extends Service {
+      replaceWith = replaceWithStub;
+    }
+    this.owner.register('service:router', RouterStub);
+    this.set('meta', getMetaForPage({ pageNumber: 2 }));
```

Comment:
Si j'en crois ce qu'on a fait sur le ticket CPF, il semble maintenant qu'un simple :
`this.meta = getMetaForPage({ pageNumber: 2 });`
marche aussi bien, plus besoin du set (c'était pour du vieux Ember le set)